docs/20-guides/101-datocms/index.md

```diff
@@ -112,24 +108,25 @@ The query for that call is this:
 }
 ```
 
-Now we have all the data we need to create a repeating grid for our blog cards as well as a simple tag filter.
+This provides all necessary data to create a repeating grid for blog cards and a straightforward tag filter.
 
-We won’t go into details for the design portion of the blog but the blog card is a simple `<a>` tag wrapping the content which uses the **slug** from the API call for its href attribute.
-For the tag filter buttons, we'll repeat them based on the tags fetched in our second API call. These will set a variable used to filter the repeating blog cards via the "filterByTag" formula.
+While detailed design aspects are outside the scope of this guide, the blog card is typically an `<a>` tag wrapping content, using the **slug** from the API call for its `href` attribute.
 
-As you can see, the layout is composed mostly of two repeated elements which makes it very nice to work with and update if we need to. We could even create components for the blog card as well but this is a small project and we won’t be reusing these elements anywhere else.
-In a typical build, however, you’d most likely build these out as components.
+Tag filter buttons are repeated based on the tags fetched in the second API call. These buttons will set a variable used to filter the repeating blog cards via the "filterByTag" formula.
+
+The layout predominantly consists of two repeated elements, which makes it easy to modify. Components could also be created for the blog card, but for smaller projects where elements are not reused, this may not be strictly necessary. In typical development, however, building these as components is common practice.
```

Comment:
```suggestion
The layout consists of two repeated elements. You could also create components for the blog card, but for smaller projects where elements are not reused, this may not be necessary. In typical development, however, building these as components is common practice.
```